netsim/extra/evpn.es/plugin.py

```diff
@@ -12,27 +12,70 @@
 _requires = [ 'evpn' ]
 _execute_after = [ 'evpn', 'lag' ]
 
+# Ethernet Segments can be supported on LAG and "physical interfaces", which means the following types for netlab
+_es_supported_on = [ 'lag', # LAG interfaces
+                     'p2p', 'stub', 'lan'  # other physical interfaces
+                    ]
+
 def post_transform(topology: Box) -> None:
     global _config_name
     for node in topology.nodes.values():
         features = devices.get_device_features(node,topology.defaults)
         es_supported = 'evpn.es' in features
         if not es_supported: continue
+        # Load Ethernet Segments data and expand it
+        es_data = node.get('evpn.ethernet_segments', {})
+        for intf in node.get('interfaces',[]):
+            intf_es = intf.get('evpn.es', '')
+            if not intf_es: continue
+            if intf_es and intf.type not in _es_supported_on:
+                log.error(
+                    f'Node {node.name}({node.device}) does not support EVPN Ethernet Segment on {intf.type} interfaces (found on: {intf.ifname})',
```

Comment:
This is not specific to the device/node, it's the plugin that doesn't support it (on any device)

Reply:
Well, I may be wrong, but I don't think ES - in general - is supported on i.e. irb interface or tunnel interface